de-lib/src/main/java/org/iplantc/de/analysis/client/presenter/AnalysesPresenterImpl.java

```diff
@@ -158,13 +162,23 @@ public void onSuccess(String result) {
     AnalysesView view;
     @Inject
     FilterAutoBeanFactory filterAutoBeanFactory;
+    @Inject
+    AnalysesAutoBeanFactory factory;
+    @Inject
+    ViceLogsView viceLogsView;
 
-    private final HasHandlers eventBus;
     AnalysisPermissionFilter currentPermFilter;
     AppTypeFilter currentTypeFilter;
 
-    @Inject
-    AnalysesAutoBeanFactory factory;
+    private String viceLogsAnalysisId;
+    private String viceLogsSinceTime = "0";
+    private String viceLogsAnalysisName;
+    private Timer viceLogsFollowTimer;
+    private final HasHandlers eventBus;
+    private String baseDebugId;
+
+
+    final int VICE_LOGS_FOLLOW_INTERVAL = 10000;
```

Comment:
This might be better as a config value so we don't have to update code if this interval doesn't work out

Reply:
you mean `de.properties` ?

Reply:
Yup, `de.properties` 👍